client/rpc/src/state/mod.rs

```diff
@@ -244,6 +255,27 @@ impl<B, E, Block, RA> StateApi<Block::Hash> for State<B, E, Block, RA>
 		self.backend.storage_keys(block, key_prefix)
 	}
 
+	fn storage_keys_paged(
+		&self,
+		prefix: StorageKey,
+		count: u32,
+		start_key: Option<StorageKey>,
+		block: Option<Block::Hash>,
+	) -> FutureResult<Vec<StorageKey>> {
+		if count > STORAGE_KEYS_PAGED_MAX_COUNT {
+			return Box::new(result(Err(client_err(
```

Comment:
I want a variant for this in the `Error` enum and have a fixed error code assigned. This makes it possible for the other side to react to this without parsing string errors.

Reply:
What is the best place for me to add this new variant? Add a new `enum Error` in this file?

Reply:
Add it to the already existing error enum for state.